net-misc/spice-gtk/spice-gtk-0.33-r2.ebuild

```diff
@@ -9,11 +9,11 @@ VALA_USE_DEPEND="vapigen"
 inherit autotools eutils xdg-utils vala readme.gentoo-r1
 
 DESCRIPTION="Set of GObject and Gtk objects for connecting to Spice servers and a client GUI"
-HOMEPAGE="http://spice-space.org https://cgit.freedesktop.org/spice/spice-gtk/"
+HOMEPAGE="https://spice-space.org https://cgit.freedesktop.org/spice/spice-gtk/"
```

Comment:
Sorry, missed that one. I'll update all ebuilds pointing to spice-space.org with https://www.spice-space.org